packages/valory/skills/register_termination_abci/models.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2022 Valory AG
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""This module contains the shared state for the register-termination ABCI application."""
+
+from typing import Any
+
+from packages.valory.skills.abstract_round_abci.models import (
+    BenchmarkTool as BaseBenchmarkTool,
+)
+from packages.valory.skills.abstract_round_abci.models import Requests as BaseRequests
+from packages.valory.skills.abstract_round_abci.models import (
+    SharedState as BaseSharedState,
+)
+from packages.valory.skills.register_termination_abci.composition import (
+    RegisterTerminateAbciApp,
+)
+from packages.valory.skills.termination_abci.models import TerminationParams
+from packages.valory.skills.transaction_settlement_abci.models import (
+    RandomnessApi as TransactionSettlementRandomness,
+)
+from packages.valory.skills.transaction_settlement_abci.models import TransactionParams
```

Comment:
why are we creating a dependency on transaction_settlement_abci here?

Reply:
Not sure what you mean, we already have a dependency to `transaction_settlement_abci` because of termination.

Reply:
Ok - I'll need to see this in the base PR again once I have a full view